hack/dev/scheduler-config.yaml.template

```diff
@@ -1,4 +1,4 @@
-apiVersion: kubescheduler.config.k8s.io/v1beta2
```

Comment:
v1beta2 is already deprecated and will be removed soon:
https://github.com/kubernetes/kubernetes/issues/117556